src/styles/product.scss

```diff
@@ -10,7 +10,16 @@
     width: 160px;
     flex: 134px 0 0;
     height: 134px;
-    margin: 30px 20px 45px;
+    margin: 27px 20px 42px;
+    border: 3px solid #fff;
+    border-radius: 5px;
+    transition: transform 1s;
+    box-sizing: content-box;
+
+    &:hover {
+      transform: scale(1.2,1.2);
+      border: 3px solid #f3f3f3;
```

Comment:
Add padding so the top left and right borders of the image match the outer borders of the card. Check the pictures by the link.
https://github.com/mate-academy/layout_catalog/blob/master/transitions.md